esi_leap/common/policy.py

```diff
@@ -83,6 +83,11 @@
         'rule:is_admin or rule:is_owner',
         'Delete offer',
         [{'path': '/offers/{offer_ident}', 'method': 'DELETE'}]),
+    policy.DocumentedRuleDefault(
```

Comment:
This reminds me that esi-leap/docs/esi-leap-policy.md misses the 'esi_leap:offer:claim' policy.

Reply:
Updated!